test/fuzzing/retrieve_blocks_fuzz.cpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "block_loader_fixture.hpp"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, std::size_t size) {
+  static fuzzing::BlockLoaderFixture fixture;
+
+  if (size < 1) {
+    return 0;
+  }
+
+  iroha::network::proto::BlocksRequest request;
+  if (protobuf_mutator::libfuzzer::LoadProtoInput(true, data, size, &request)) {
+    grpc::ServerContext context;
+    MockServerWriter writer;
```

Comment:
ServerWriter cannot be mocked :( I've tried with ServerWriterInterface, but it cannot be casted safely (e.g only via `reinterpret_cast`. Here's an example of such mock (path _test/module/vendor/grpc/grpc_mocks.hpp_):

```cpp
class MockServerWriter
    : public grpc::ServerWriterInterface<iroha::protocol::ToriiResponse> {
  MOCK_METHOD1(Write, void(iroha::protocol::ToriiResponse));
  MOCK_METHOD2(Write,
               bool(const iroha::protocol::ToriiResponse &,
                    grpc::WriteOptions));
  MOCK_METHOD0(SendInitialMetadata, void());
  MOCK_METHOD1(NextMessageSize, bool(uint32_t *));
};
```

But it's not really a good idea to use it. Need to think about that issue